TalkableSDK.podspec

```diff
@@ -24,8 +24,8 @@ Pod::Spec.new do |spec|
   spec.platform     = :ios, "9.0"
 
   spec.source       = { :git => "https://github.com/talkable/ios-sdk.git", :tag => "#{spec.version}" }
-  spec.source_files  = "Talkable/**/*.{h,m}", "3rdParty/**/*.{h,m}"
-
+  spec.source_files  = "Talkable/**/*.{h,m}"
+  spec.dependency 'AFNetworking', '~> 2.6.3'
```

Comment:
Shouldn't we remove `3rdParty` folder after this change?

Reply:
Because of linking problem. In 3rdParty folder was AFNetworking, and when project compiling, AFNetworking files links with files from AFNetworking pod. This caused an error. So I just added dependency to AFNetworking in pod.spec file

Reply:
For history: we leaving `3rdParty` folder as there is a possibility to build the SDK manually which will not work without this folder